### ISSUE TITLE
I can't find darknet.exe in C:\darknet\build\darknet\x64

### ISSUE THREAD

**Issue by brian208579** [AlexeyAB/darknet#4667]
Hi @AlexeyAB 
I followed your steps to compile with cmake-gui
I can't find **darknet.exe** in **C:\darknet\build\darknet\x64**
It exists **C:\darknet\Release**
Is it correct ?
Thanks ...

**Comment by ajbot2019** [comment]
@brian208579  yes the darket.exe file will be present in the folder called darketnet itself .

**Comment by brian208579** [comment]
@ajbot2019 
Thanks for your response 
So ... I need to copy the C:\darknet\Release\darknet.exe
to C:\darknet ？
beacuse when I compile success , the darknet.exe doesn’t exist in C:\darknet

**Comment by ajbot2019** [comment]
@brian208579   check the  working directory where you issued the command to complie ,the darknet.exe file fill be present in same path .

**Comment by brian208579** [comment]
@ajbot2019  
Good, thank you very much
